master/reference/calicoctl/resources/policy.md

```diff
@@ -65,6 +65,52 @@ the policy is enforced after connection tracking and any DNAT.
 See [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal)
 for how `doNotTrack` and `preDNAT` can be useful for host endpoints.
 
+The `types` field is useful when a policy specifies no ingress rules and you
+want that to mean that no ingress should be allowed, to each endpoint to which
+that policy applies, except as allowed by any other policies that apply to that
+endpoint.  You can express that by including `ingress` as one of the values of
+the `types` field - effectively saying "please apply this policy to ingress
+traffic, even though it looks like it should not."
+
+Similarly, when a policy specifies no egress rules and you want that to mean
+that no egress should be allowed ..., you can express that by including
+`egress` as one of the values of the `types` field.
+
```

Comment:
@lxpollitt Perhaps I am misunderstanding you here, but it seems to me that both of your points are wide of the mark, as:

1. Those two paragraphs are precisely about the cases where a user _does_ set the `types` field.  The _following_ paragraph covers how `types` is defaulted when not explicitly set.  In other words, I believe the doc is already organized exactly as your comment suggests.

2. We have agreed to default the `types` field in a way that _doesn't_ give us back-compatibility with Calico <2.6, and such that the behaviour is intuitive when `types` is _not_ specified.